Makefile

```diff
@@ -23,11 +23,12 @@ API_PKG := $(GO_PKG)/api
 ROOT_PKG := github.com/openshift/operator-framework-olm
 
 PSM := $(addprefix bin/, psm)
+COLLECT_PROFILES_CMD := $(addprefix bin/, collect-profiles)
 OPM := $(addprefix bin/, opm)
 OLM_CMDS  := $(shell go list -mod=vendor $(OLM_PKG)/cmd/...)
-PSM_CMDS := $(shell go list -mod=vendor github.com/openshift/operator-framework-olm/cmd/...)
+PSM_CMDS := $(shell go list -mod=vendor github.com/openshift/operator-framework-olm/cmd/package-server-manager)
 REGISTRY_CMDS  := $(addprefix bin/, $(shell ls staging/operator-registry/cmd | grep -v opm))
-
+OPENSHIFT_CMDS := $(shell go list -mod=vendor github.com/openshift/operator-framework-olm/cmd/collect-profiles)
```

Comment:
I think we may want to go in a different direction here and generalize this further. How do we feel about consolidating both of these downstream binaries using something like `ROOT_CMDS` and building using a single target?

The `build/olm` target would remove things like `$(PSM_CMDS)` and `$(OPENSHIFT_CMDS)` and instead just specify `$(MAKE) $(ROOT_CMDS) $(OLM_CMDS)`. Note: there may be a better name for this than root cmd.

Reply:
I agree with you in principle, but the command we've generated should be removed when OpenShift defines its continuous profiling strategy in the future. Given our goals to deprecate this command, could we leave it as its own binary while committing to moving the package-server command to a subcommand of a new binary (in a separate PR)?